### ISSUE TITLE
What's the better loop json syntax?

### ISSUE THREAD

**Issue by MiniXC** [MEH-Design/frix#41]
Is this
````json
"someLoop" : [
  {
    "type": "someType",
    "data": "element"
  }
]
````
or this
````json
"someLoop" : [
  {
    "someType": "element"
  }
]
````
better?

**Comment by MiniXC** [comment]
The first syntax is preferred.

**Comment by MiniXC** [comment]
Fixed tests in 62333ba0dddb385e4433f49caf7c732285dd0b53
